### PR TITLE
Update generated component reference docs for 1.22

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by tengqm** [comment]
@PI-Victor

File: content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ kube-controller-manager [flags]
 <td colspan="2">--cluster-signing-duration duration&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 8760h0m0s</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>The length of duration signed certificates will be given.</p></td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>The max length of duration signed certificates will be given.  Individual CSRs may request shorter certs by setting spec.expirationSeconds.</p></td>
```
Comment (jihoon-seo) [inline content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md:211]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>The max length of duration signed certificates will be given.  Individual CSRs may request shorter certs by setting `spec.expirationSeconds`.</p></td>
```

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Generate keys and certificate signing requests
 
 Generates keys and certificate signing requests (CSRs) for all the certificates required to run the control plane. This command also generates partial kubeconfig files with private key data in the  "users &gt; user &gt; client-key-data" field, and for each kubeconfig file an accompanying ".csr" file is created.
 
-This command is designed for use in [Kubeadm External CA Mode](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). It generates CSRs which you can then submit to your external certificate authority for signing.
+This command is designed for use in [Kubeadm External CA Mode](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). It generates CSRs which you can then submit to your external certificate authority for signing.
```
Comment (jihoon-seo) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md:20]:
```suggestion
This command is designed for use in [Kubeadm External CA Mode](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). It generates CSRs which you can then submit to your external certificate authority for signing.
```

**Review by jihoon-seo — COMMENTED** [review]
@tengqm PTAL! 😊
(I guess that these files are generated ones and you have re-generated and updated them.)

**Comment by sftim** [comment]
/retitle Update generated component reference docs for 1.22
/approve

File: content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md:211]: We don't (yet) have a good way to put Markdown or other formatting in the source code that these are generated from.

**Comment by neolit123** [comment]
These are generated files, though. If you think minor changes like quoting
have to made, it has to be a PR for kubernetes/kubernetes first.

**Comment by gjkim42** [comment]
It seems that `content/en/docs/reference/command-line-tools-reference/kubelet.md` is not generated.
Am I missing something?

**Comment by neolit123** [comment]
Some of the review was for generated files
https://github.com/kubernetes/website/pull/29242#discussion_r683122974

Some files are indeed aurhored content.

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md:20]: I think https://github.com/kubernetes/website/pull/28234 should not have overwritten this content; it is marked as autogenerated.

**Comment by sftim** [comment]
This might well be a partial fix for  #29255; I think we'd also need to do `kubectl` and the kubelet; perhaps as separate PRs.

**Comment by annajung** [comment]
Thanks @tengqm 
/lgtm

@PI-Victor would be good to follow up with Qiming (if haven't already) on how we can improve this process for future releases 

cc @savitharaghunathan @reylejano

**Pull request merged**